src/network/rpc/Test.cpp

```diff
@@ -329,15 +329,16 @@ Json::Value Test::get_votes(const Json::Value & /*param1*/) {
   Json::Value res;
   try {
     if (auto node = full_node_.lock()) {
-      std::shared_ptr<VoteManager> vote_mgr = node->getVoteManager();
+      auto db = node->getDB();
 
-      auto verified_votes = vote_mgr->getVerifiedVotes();
-      auto unverified_votes = vote_mgr->getUnverifiedVotes();
+      auto verified_votes = db->getVerifiedVotes();
```

Comment:
I do not understand why we are now get all votes from DB and not from vote_mgr. DB is much slower as vote_mgr keeps everything in memory

Reply:
Since we don't keep verified votes in vote_mgr anymore, and just keep verified votes hash in vote_mgr for saving memory

Reply:
Move back verified votes in memory, fixed